templates/_helpers.yaml

```diff
@@ -70,12 +76,24 @@
         name: {{ $config.secretName }}
         key: {{ default "value" $config.secretKey }}
   {{- end }}
-  {{- if eq .Values.executor "KubernetesExecutor" }}
-    {{- range $i, $config := .Values.secret }}
-  - name: AIRFLOW__KUBERNETES_SECRETS__{{ $config.envName }}
-    value: {{ printf "%s=%s" $config.secretName $config.secretKey }}
```

Comment:
@kushalmalani we don't need to delete this chunk right? If someone is using the old method with `.Values.secret`, they'd still want those envs to get set into the dynamically launched worker pods.

Reply:
oh yeah. but this will redundant adding of env vars in the new mode. Should i take this out into a separate helper for the old way only?

Reply:
Yea def is redundant, just thinking about backward compatibility. In my other comment I suggest combining both "old ways" (`Values.secret`, and `Values.env`) into a helper and adjusting the logic to allow for either the new way, or the old way, but not both at the same time. Essentially the new way will disable any of the old stuff.